gcexport.py

```diff
@@ -457,7 +457,7 @@ def parse_arguments(argv):
     Setup the argument parser and parse the command line arguments.
     """
     current_date = datetime.now().strftime('%Y-%m-%d')
-    activities_directory = './' + current_date + '_garmin_connect_export'
```

Comment:
Is this `./` cross-platform?

Reply:
As far as I know Python does the right thing in all operating systems when using a slash.

Reply:
Thanks! Yeah, it seems more like a Windows feature but probably works well.